tests/Serializer/PayloadSerializerTest.php

```diff
@@ -54,7 +55,7 @@ public function testSerialize(Event $event, string $expectedResult, bool $isOutp
 
         $result = $this->serializer->serialize($event);
 
-        if (EventType::transaction() !== $event->getType()) {
+        if (EventType::transaction() !== $event->getType() && !$event->hasAttachments()) {
```

Comment:
This will break old self-hosted Sentry installations without Relay when using attachments. This might be odd in terms of UX, as we can't make assumptions about the destination we sent the events to.